crates/core/c8y_smartrest/Cargo.toml

```diff
@@ -5,11 +5,11 @@ authors = ["thin-edge.io team <info@thin-edge.io>"]
 edition = "2018"
 
 [dependencies]
+agent_mapper_interface = { path = "../agent_mapper_interface" }
```

Comment:
This dependency is really suspicious. A crate that is related to SmartRest should be unrelated to the agent.

* This is due to the `SmartRestSerializerError::UnsupportedOperationStatus` enum.
* This error case has to be replaced with `UnsupportedOperationStatus { status: String }`.

Reply:
it also requires even OperationStatus